internal/opslevel/opslevel.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+// Description: See package comment for this one-file package.
+
+// Package opslevel is a package meant to help utilize the OpsLevel API in GitHub actions.
+package opslevel
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/getoutreach/actions/internal/slack"
+	"github.com/opslevel/opslevel-go/v2022"
+	"github.com/pkg/errors"
+)
+
+// NewClient returns a new opslevel client configured with the token we expect to exist in
+// the environment.
+func NewClient() (*opslevel.Client, error) {
+	opslevelToken := strings.TrimSpace(os.Getenv("OPSLEVEL_TOKEN"))
+	if opslevelToken == "" {
+		return nil, errors.New("OPSLEVEL_TOKEN environment variable is empty")
+	}
+	return opslevel.NewGQLClient(opslevel.SetAPIToken(opslevelToken)), nil
+}
+
+// LifecycleToLevel maps lifecycle index to level index
+// we want to keep this at the index level in case names or other attributes change
```

Comment:
Formatting (fix run-on):
```suggestion
// LifecycleToLevel maps lifecycle index to level index.
// We want to keep this at the index level in case names or other attributes change.
```